src/client/app/+home/home.component.ts

```diff
@@ -1,12 +1,12 @@
 import { FORM_DIRECTIVES } from '@angular/common';
 import { Component } from '@angular/core';
 
-import { NameListService } from '../../shared/index';
+import { NameListService } from '../shared/index';
 
 @Component({
   selector: 'sd-home',
-  templateUrl: 'app/+home/components/home.component.html',
-  styleUrls: ['app/+home/components/home.component.css'],
+  templateUrl: 'app/+home/home.component.html',
```

Comment:
Hey @ludohenin, i just stumbled across this recent talk https://youtu.be/WAPQF_GA7Qg?t=5m39s by John Papa and saw that the moduleId: module.id is explicitly included within the snippet of Visual Studio Code and he furthermore explains that Angular2 makes use of this since it uses commonjs. Sorry if i got something wrong there, but doesn't that open the possiblity of re-adding module.id, considering the seed uses commonjs as well? 
Thanks for your input :)

//cc @mgechev